src/components/Atoms/Controls/Input/index.tsx

```
@@ -1,61 +1,52 @@
 import { IoEye } from "react-icons/io5";
-import { Tooltip } from "../../Misc/Tooltip";
-import { AiOutlineInfoCircle } from "react-icons/ai";
-
-interface CustomChangeEvent {
+import { InputLabel } from "./components/InputLabel";
+import { InputError } from "./components/InputError";
+import DefaultInput from "./components/DefaultInput";
+export interface CustomChangeEvent {
   target: {
     name: string;
     value: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)